latest/base.yml

```diff
@@ -9,7 +9,7 @@ playbooks_version: main
 osism_projects:
   ara: '1.5.7'
   # renovate: datasource=github-releases depName=moby/moby
-  docker: '5:20.10.13'
+  docker: '5:v20.10.14'
```

Comment:
The leading v has to be removed.

Reply:
See https://github.com/osism/release/issues/352